app/views/events/show.html.erb

```diff
@@ -35,6 +35,12 @@
         %>
       </div>
     </div>
+    <div class="gi gi__1-3">
+      <h3 class="h4 udl">Event organiser</h3>
+      <div class="small">
+        <span><%= link_to @event.partner, @event.partner %></span>
```

Comment:
do events always have partners?

Reply:
yes!

Reply:
can't have an an event without a calendar, can't have a calendar without a partner

Reply:
that makes sense

Reply:
that's a bit weird: the relation is optional but we have a validation for its presence?

Reply:
🤷‍♀️